### PR TITLE
Spatial filtering - add a new checkbox to filter by users and not by …

### PR DESCRIPTION
**Pull request by mdouchin**
* **Description**: Spatial filtering (by polygon intersection): add a new checkbox "Filter by user" to filter by users and not by user groups
* **Funded by**: Valabre

Related to https://github.com/3liz/qgis-lizmap-server-plugin/pull/10

### CODE REVIEW

**Review by rldhont — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Spatial filtering: add a new checkbox "Filter by user" to filter by users and not by user groups
```
Comment (Gustry) [inline CHANGELOG.md:5]: Add a note for LWC 3.6

**Review by Gustry — CHANGES_REQUESTED** [review]
Please add the checkbox in the main python file, at the beginning, there is a list of QWidgets for each LWC version. This will make the blue color if LWC < 3.5 is selected.

File: CHANGELOG.md (hunk 1 above)
Comment (mdouchin) [reply CHANGELOG.md:5]: NB: this will also work with LWC 3.5. I will add a comment in the `CHANGELOG.md`

**Comment by mdouchin** [comment]
> Please add the checkbox in the main python file, at the beginning, there is a list of QWidgets for each LWC version. This will make the blue color if LWC < 3.5 is selected.

Done ! 
I also changed the wording and use `filter_by_user` instead of `filter_private`.
This PR is now ready for merge if you agree

**Review by Gustry — APPROVED** [review]
LGTM, thanks

**Pull request merged**